README.md

```diff
@@ -135,6 +135,48 @@ This is more a hack. The chart usually tries to infer its own dimensions regardi
 <linechart width="150" height="100"></linechart>
 ```
 
+#### Custom events
+We can attach event handlers for *click*, *hover* and *focus* events of the chart.
+
+##### Click
+The event handler for the *click* event get's triggered when the mouse clicks on a dot or column of the chart in tooltip modes *none* and *axes*.
+
+```js
+$scope.onClick = function(d, i){
+  conosole.log(d, i);
```

Comment:
Unfortunately, `conosole` is not well-supported. Let's use `console` instead, if you don't mind :-)